### ISSUE TITLE
Reader lag during page transition 

### ISSUE THREAD

**Issue by Gtoonm** [mihonapp/mihon#700]
### Steps to reproduce

Open any chapter 
Swipe left or right to change to the next/previous page

### Expected behavior

Page changes should be smooth when reading any chapter. Wether it's downloaded or not

It should be a simple smooth animation when going from page to page

### Actual behavior

There is a noticable choppiness and lag when changing pages in any manga and manwha. This is not a device specific problem as I have tested this on separate devices as well as the last release of tachiyomi and the issue is not present in it.

### Mihon version

0.16.5

### Android version

Android 14

### Device

ROG phone 8 pro

### Other details

This only appeared after the 0.16.4 updates 

0.16.3 worked correctly 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Xori71** [comment]
Yup, I have been facing this since the day I started using Tachi. For obvious reasons it has carried over to Mihon.

**Comment by Aryanblood17** [comment]
I encountered the same problem multiple times today but I don't remember ever facing these issues with Tachi or even Mihon before today

**Comment by plumber-craic** [comment]
This seems solved for me in beta r6903
